src/web/routes/application/check/get.js

```diff
@@ -8,7 +8,8 @@ const pageContent = ({ translate }) => ({
   sendApplicationHeader: translate('check.sendApplicationHeader'),
   sendApplicationText: translate('check.sendApplicationText'),
   buttonText: translate('buttons:acceptAndSend'),
-  changeText: translate('check.change')
+  changeText: translate('check.change'),
+  previous: steps[steps.length - 1].path // last step
```

Comment:
This isn't a safe way to access the `path` property - if steps was empty or undefined, or if the step didn't have a path property for some reason then this would throw an error

Reply:
Updated to use ramda path and check existence of steps before calling .length()